audb/core/load.py

```diff
@@ -64,7 +66,7 @@ def _cached_versions(
                 0,
                 (
                     audeer.LooseVersion(row['version']),
-                    flavor_root,
+                    str(flavor_root),
```

Comment:
When can it happen that this is not a string? When it starts with a digit?

Reply:
It's just a type hint issue because `df.iterrows()` returns `(Hashable, Series`).

Reply:
OK